docs/docs/04-features.md

```diff
@@ -308,3 +308,20 @@ Note that `@snowpack/plugin-optimize` will optimize your build, but won't bundle
   "plugins": [["@snowpack/plugin-webpack", {/* ... */}]]
 }
 ```
+
+### Installing Non-JS Packages
+
+When installing packages from npm, You may encounter some non-JS code that can only run with additional parsing/processing. Svelte packages, for example, commonly include `.svelte` files that will require additional tooling to parse and install for the browser.
```

Comment:
one thing to add: the reason that we removed this was that we add common ones for you via plugin. plugin-svelte, for example, adds this for you automatically to handle Svelte packages.

The section is still good though, but maybe moved down to troubleshooting now that its less commonly used. Also, you can add a note that svelte & vue plugins do this for you automatically.